jemalloc-sys/build.rs

```diff
@@ -77,6 +77,21 @@ fn main() {
         warning!("jemalloc support for `{}` is untested", target);
     }
 
+    let mut use_prefix =
+        env::var("CARGO_FEATURE_UNPREFIXED_MALLOC_ON_SUPPORTED_PLATFORMS").is_err();
+
+    if !use_prefix && NO_UNPREFIXED_MALLOC.iter().any(|i| target.contains(i)) {
+        warning!(
+            "Unprefixed `malloc` requested on unsupported platform `{}` => using prefixed `malloc`",
+            target
+        );
+        use_prefix = true;
+    }
+
+    if use_prefix {
+        println!("cargo:rustc-cfg=prefixed");
+    }
+
```

Comment:
What's the reason for this change?

Reply:
On systems where jemalloc must be prefixed:
1. the prebuilt jemalloc has to be prefixed
2. `link_name` should be adapt to this. So the `rustc-cfg=prefixed` should be emitted

Reply:
Sure, but IIUC this commit just moved the lines around, without any change to functionality.

Reply:
```println!("cargo:rustc-cfg=prefixed");```

has to occur before the early ```return``` in L100 when ```JEMALLOC_OVERRIDE``` is set

Reply:
Ah indeed, thanks. So could you add a comment about this ? Otherwise this LGTM.

Reply:
Comment added 😃